soc/riscv/riscv-privilege/hpmicro/Kconfig.defconfig.series

```diff
@@ -0,0 +1,39 @@
+# Copyright (c) 2022 HPMicro
+# SPDX-License-Identifier: Apache-2.0
+
+if SOC_SERIES_HPMICRO
+
+source "soc/riscv/riscv-privilege/hpmicro/Kconfig.defconfig.hpm6*"
+
+config SOC_SERIES
+	default "hpmicro"
+
+config SYS_CLOCK_HW_CYCLES_PER_SEC
+	default 24000000
```

Comment:
MCHTIMR clock souce is CLK24MHZ, but the default value is 1MHZ.
I have tested if SYS_CLOCK_HW_CYCLES_PER_SEC is 24MHZ, the k_time 1s is 2.4s.
it need change to the default 1MHZ

Reply:
Thanks for the feedback. The program is designed for 24Mhz, so I will add code to modify the clock division to 1.